src/documentation/pages/components/modal/index.vue

```diff
@@ -0,0 +1,36 @@
+<template lang="pug">
+  <Wrapper>
```

Comment:
Apparently adding <Wrapper> to the root-most index.vue breaks navigation :( so we should exceptionally use <div> here instead